README.md

```diff
@@ -21,6 +21,153 @@ as these things will be provided by your cloud provider.
+          app.kubernetes.io/name: nginx-errors
+          app.kubernetes.io/part-of: ingress-nginx
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: nginx-errors
+            app.kubernetes.io/part-of: ingress-nginx
+        spec:
+          containers:
+          - name: nginx-error-server
+            # Update the image if there is a new version available
+            image: registry.k8s.io/ingress-nginx/custom-error-pages:v1.0.2@sha256:b2259cf6bfda813548a64bded551b1854cb600c4f095738b49b4c5cdf8ab9d21
+            ports:
+            - containerPort: 8080
+            # Mounting custom error page from ConfigMap 1
+            volumeMounts:
+            - name: custom-error-pages-404
+              mountPath: /www/404.html
+              subPath: 404.html
+            # Mounting custom error page from ConfigMap 2
+            - name: custom-error-pages-503
+              mountPath: /www/503.html
+              subPath: 503.html
+
+          # Mounting volumes from two ConfigMaps
+          volumes:
+          - name: custom-error-pages-404
+            configMap:
+              name: custom-error-pages-404
+              items:
+              - key: "404"
+                path: "404.html"
+          - name: custom-error-pages-503
+            configMap:
+              name: custom-error-pages-503
+              items:
+              - key: "503"
+                path: "503.html"
+    ---
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      annotations:
+        nginx.ingress.kubernetes.io/custom-http-errors: 503,404
+        nginx.ingress.kubernetes.io/default-backend: nginx-errors
+      name: wildcard-test-srv-dev
+      namespace: serve-dev
+    spec:
+      defaultBackend:
+        service:
+          name: nginx-errors
+          port:
+            number: 80
+      rules:
+      # Change this if you are using a different domain
+      - host: '*.serve.scilifelab.se'
+        http:
+          paths:
+          - backend:
+              service:
+                name: nginx-errors
+                port:
+                  number: 80
+            path: /404.html
+            pathType: ImplementationSpecific
+      tls:
+      - hosts:
+        # Change this if you are using a different domain
+        - '*.serve.scilifelab.se'
+        secretName: prod-ingress
+    ---
+    # Custom error page configMap for the 404 error
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: custom-error-pages-404
+    data:
+      "404": "Error"
+    ---
+    # Custom error page configMap for the 503 error
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: custom-error-pages-503
+    data:
+      "503": "Error"
+    ```
+</details>
+
+Apply the configuration to the namespace:
+```bash
+$ kubectl apply -f custom-error-backend.yaml
+```
+
+In the rancher dashboard change the values for configmaps `custom-error-pages-404` and `custom-error-pages-503` to 
```

Comment:
I would advice against using Rancher's dashboard features to edit ConfigMaps. Not that it would not work, just that it's very easy to make mistakes. Instead, I suggest you create the ConfigMaps using `kubectl`.

Reply:
Should be possible to do like this:

```shell
kubectl create configmap custom-error-pages --from-file 404=./html/404.html --from-file 503=./html/503.html
```

The result should look similar to this: https://github.com/kubernetes/ingress-nginx/blob/6ceccbd67b140b7626670ad17f926f121a9e5563/docs/examples/customization/custom-errors/custom-default-backend-error_pages.configMap.yaml